html/index.html

```diff
@@ -698,8 +700,15 @@ <h2>Filter</h2>
                     <label for="cloud-plugins-installed">Installed only</label>
                 </div>
                 <div class="control-group mod-hidden">
-                    <input type="checkbox" name="unstable" id="cloud-plugins-stable" />
-                    <label for="cloud-plugins-stable">Show beta plugins</label>
+                    <div class="switch" id="cloud-plugins-stable">
```

Comment:
We should rename a couple of ids here.
`cloud-plugins-stable` no longer makes sense, it is more of a `cloud-plugins-source`.

Then, using small ids such as `official` and `labs` seems wrong.
We are taking this global id just for the plugin store, something more verbose would be nice.
I suggest `cloud-radio-official` and `cloud-radio-labs`.

Reply:
@falkTX should we also change the preferences key `show-unstable-plugins` into `show-labs-plugins` or `show-plugins-source` for instance?

Reply:
Ah yes, `show-labs-plugins` seems to go into the right direction.

Reply:
looking into that, I've just also noticed this: https://github.com/moddevices/mod-ui/blob/labs/html/js/desktop.js#L831-L834
how to handle this now?

Reply:
The message needs to change to mention Labs instead unstable plugins.
We can ask Jesse for the exact wording.